server/src/main/java/org/opensearch/throttling/admissioncontrol/enums/TransportActionType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.throttling.admissioncontrol.enums;
+
+import java.util.Locale;
+
+/**
+ * Enums that defines the type of the transport requests
+ */
+public enum TransportActionType {
```

Comment:
Can we have one class which contains both enum and the action map

Reply:
For now keeping this as different both actionsMap and Enum. Will update in next revisions if this ENUM has limited scope and can be fit into actionsMap class.